pages/security.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Security
+main_nav_id: security
+---
+
+# Kotlin Security
+
+We do our best to make sure our products are free of security vulnerabilities. To reduce the risk of introducing a vulnerability,
+you can follow these best practices: 
+
+* Always use the latest Kotlin release. For security purposes, we sign our releases published on [Maven Central](https://search.maven.org/search?q=g:org.jetbrains.kotlin) 
+with these PGP keys.
+* Use the latest versions of your application’s dependencies. If you still need to use a specific dependency version, 
+periodically check if any new security vulnerabilities have been discovered. You can follow 
+[the guidelines from GitHub](https://help.github.com/en/github/managing-security-vulnerabilities/managing-vulnerabilities-in-your-projects-dependencies) 
+or browse known vulnerabilities in the [CVE base](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=kotlin).
+
+We are very eager and grateful to hear about any security issues you find. To report vulnerabilities that you discover in Kotlin,
+please post a message directly to our [issue tracker](https://youtrack.jetbrains.com/issues/KT) (setting the issue type to ['Security Problem'](https://youtrack.jetbrains.com/issues/KT?q=%23%7BSecurity%20Problem%7D%20)) 
```

Comment:
```suggestion
please file an issue to our [issue tracker](https://youtrack.jetbrains.com/issues/KT) (setting the issue type to ['Security Problem'](https://youtrack.jetbrains.com/issues/KT?q=%23%7BSecurity%20Problem%7D%20)) 
```
`message to issue tracker` sounds a bit awkward to me.

Reply:
It is standard wording for security problems in JB. :) Don't know why.